app/controllers/concerns/defendant_searchable.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
```

Comment:
@jsugarman, I removed this file in https://github.com/ministryofjustice/laa-court-data-ui/pull/231, we shouldn't need it any more. I don't know whether removing it from your pr would improve code coverage?

Reply:
ooh, odd

Reply:
removed